lib/src/main/java/org/team100/lib/util/CameraAngles.java

```diff
@@ -52,61 +33,56 @@ public CameraAngles(
      * and sim
      */
     public CameraAngles() {
-        m_downwardAngleDegrees = 0;
-        m_horzFOVDegrees = 0;
-        m_vertFOVDegrees = 0;
-        m_horzResolution = 0;
-        m_vertResolution = 0;
+        m_downwardAngleRads = 0;
         m_cameraHeightMeters = 0;
         m_xOffset = 0;
         m_yOffset = 0;
     }
 
     /**
+     *         0 should be center of FOV
      * @return A robot relative translational x value of an object in a camera in
      *         meters
      */
-    public double getX(double vertPixels) {
+    public double getX(double vertFOVRads) {
```

Comment:
i think you mean this argument to be the rotation to the target, it doesn't have anything to do with "FOV" so why is it called that?  i think you should clarify what this represents.